### ISSUE TITLE
 Using my project externally is also an important part missing from the official guide

### ISSUE THREAD

**Issue by renjiali68666** [supabase/supabase#21002]
This is also an important part missing from the official guidelines. At this point, you can access Kong Studio http://localhost:8000/ instrument panel http://localhost:3000 But these cannot be accessed via HTTPS and can only be accessed on the host. The most commonly used solution is to point your domain to the public IP address of the host, and then use Nginx to reverse proxy to Kong or Studio. This doesn't work for me

**Comment by encima** [comment]
Thanks for opening! Is this a request for a docs update? You can set the URL using `SUPABASE_PUBLIC_URL` and then add a proxy through to the studio as you said; what part would you like help with?

**Comment by renjiali68666** [comment]
> 感谢您的开通！这是文档更新的请求吗？您可以使用设置 URL `SUPABASE_PUBLIC_URL`，然后按照您所说的将代理添加到工作室；您需要哪方面的帮助？

Yes, but this is not possible  ，  Could you please supplement the document

**Comment by komali2** [comment]
> > 感谢您的开通！这是文档更新的请求吗？您可以使用设置 URL `SUPABASE_PUBLIC_URL`，然后按照您所说的将代理添加到工作室；您需要哪方面的帮助？
> 
> Yes, but this is not possible ， Could you please supplement the document

This is possible, I have already tested this with a traditional reverse proxy via nginx, as well as using cloudflare tunnels. 

I feel there's some shiftiness around the fact that it's only one project and if you want multiple users, you have to tediously add them to a yml file (with plaintext passwords!!!), but, it does work as a web-visible deployment, including all features. We have tested this at our co-op. 

See https://supabase.com/docs/guides/self-hosting/docker#accessing-the-apis and https://supabase.com/docs/guides/self-hosting/docker#accessing-supabase-dashboard . That gives you the ports you need to point reverse proxy at. I believe teaching people how to, in general, route ports and do reverse proxies is outside the scope of the documentation here. 

這是可能的，我已經透過 nginx 使用傳統的反向代理以及使用 cloudflare 隧道對此進行了測試。

我覺得這只是一個項目，如果你想要多個用戶，你必須繁瑣地將它們添加到 yml 檔案中（使用明文密碼！！！），但是，它確實可以作為網路可見的部署，包括所有功能。 我們已經在合作社對此進行了測試。

請參閱 https://supabase.com/docs/guides/self-hosting/docker#accessing-the-apis 和 https://supabase.com/docs/guides/self-hosting/docker#accessing-supabase-dashboard 。 這為您提供了指向反向代理所需的連接埠。 我認為教人們如何路由連接埠和進行反向代理超出了此處文件的範圍。

**Comment by renjiali68666** [comment]
> > > 感谢您的开通！这是文档更新的请求吗？您可以使用设置 URL `SUPABASE_PUBLIC_URL`，然后按照您所说的将代理添加到工作室；您需要哪方面的帮助？
> > 
> > 
> > 是的，但是这是不可能的，您能补充一下文档吗
> 
> 这是可能的，我已经通过 nginx 使用传统的反向代理以及使用 cloudflare 隧道对此进行了测试。
> 
> 我觉得这只是一个项目，如果你想要多个用户，你必须繁琐地将它们添加到 yml 文件中（使用明文密码！！！），但是，它确实可以作为网络可见的部署，包括所有功能。我们已经在合作社对此进行了测试。
> 
> 请参阅https://supabase.com/docs/guides/self-hosting/docker#accessing-the-apis和https://supabase.com/docs/guides/self-hosting/docker#accessing-supabase-dashboard。这为您提供了指向反向代理所需的端口。我认为教人们如何路由端口和进行反向代理超出了此处文档的范围。
> 
> 这是可能的，我已经使用 nginx 传统的反向代理以及使用 cloudflare 隧道对此进行了测试。
> 
> 我觉得这只是一个项目，如果你想要多个用户，你必须繁琐将它们添加到 yml 档案中（使用明文密码！！！），但是，它确实可以作为网路可见的部署，包括所有功能。我们已经在合作公司对此进行了测试。
> 
> 请参阅https://supabase.com/docs/guides/self-hosting/docker#accessing-the-apis和https://supabase.com/docs/guides/self-hosting/docker#accessing-supabase-dashboard。这为您提供了指向反向代理所需的连接端口。我认为教人们如何路由连接端口并进行反向代理超出了此处文件的范围。

非常抱歉，我在中国大陆，cloudflare 隧道虽然是可以使用的，但是这是十分不稳当。因此我需要，传统的反向代理。但是有以下问题
我配置了公网ip，但是，在8000页面下，我无法复制存储地址，而且，也无法上传，就直接报错了
I'm very sorry. I can use the cloudflare tunnel in Chinese Mainland, but it is very unstable. Therefore, I need traditional reverse proxies. But there are the following issues

I have configured a public IP address, but on the 8000 page, I am unable to copy the storage address and upload it, resulting in an error message
